### ISSUE TITLE
can't create folder inside zip

### ISSUE THREAD

**Issue by mikeyc7m** [alchemy-fr/Zippy#93]
When i try to add a file to the root of the archive it works. But when i try to add a file to a folder inside the archive it fails.

`$filePathArray["filename.jpg"] = "/some/path/to/filename.jpg"` OK
`$filePathArray["folder/filename.jpg"] = "/some/path/to/filename.jpg"` FAIL
`$archive = $zippy->create($zipFilePath, $filePathArray);`

**Comment by mikeyc7m** [comment]
hmm it works on the remote unix server but not my local windows machine.

**Comment by aztech-dev** [comment]
Hello,

Thanks for your report. I was able to reproduce your issue with the following error message:

```
Could not write to C:\Users\ADMINI~1\AppData\Local\Temp\2/zippy_5694e1f90b2a9/folder/filename.jpg
```

**Comment by aztech-dev** [comment]
Issue is fixed, you need to update to version 0.3.2 to get the fix.
